app/src/main/java/com/samco/trackandgraph/graphstatview/decorators/GraphStatLineGraphDecorator.kt

```diff
@@ -111,9 +111,9 @@ class GraphStatLineGraphDecorator(listMode: Boolean) :
 
     private suspend fun setLineGraphBounds() {
         val bounds = data!!.bounds
-        if (data!!.yRangeType == YRangeType.FIXED) {
```

Comment:
I wasn't aware of this side effect. Making it dependent on the listView sounds good to me.

Reply:
**So do you also want to not hard-set bounds in fullscreen-mode when using the fixed mode?** I feel that if you choose to view it fullscreen, you might want to scroll and not be hindered by the bounds, even though you set them (by using fixed mode).

Reply:
It's kind of tricky actually because sometimes it's more irritating that the bounds move because the lines jump around and it can be confusing. I find It depends on the nature of the data you're looking at. I think ideally you should be able to turn this on and off in the full screen view but for now it may be best to keep the behaviour as is and have it only work if the bounds are dynamic mode.